site/assets/scss/_subnav.scss

```diff
@@ -1,7 +1,7 @@
 .bd-subnavbar {
   // The position and z-index are needed for the dropdown to stay on top of the content
   position: relative;
-  z-index: $zindex-sticky;
+  z-index: 1080;
```

Comment:
I'd rather change the variable than have a hardcoded magic number :)

Reply:
@XhmikosR I added a number, because there wasn't any variable specifying **1080**, and do we really need to specify a variable because in current situation we just have to use it in one place.

Reply:
This file is used for core bootstrap(https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L830-L842), so do we need to make changes in this file as it is used for core bootstrap.
But we do have one more variable file which we can use for adding our variable for documentation and here is the [path](https://github.com/twbs/bootstrap/blob/main/site/assets/scss/_variables.scss). In my opinion we can make changes in the second file. :)

Reply:
The docs are using the core variables file too.

Anyway, we'll see how we are going to handle this.

Reply:
@XhmikosR so should I make a commit with changes or should I just leave it for time being until a decision comes up?

Reply:
I'd go for a new variable in our docs assets :)